packages/admin/resources/views/components/layouts/app.blade.php

```diff
@@ -3,7 +3,7 @@
 ])
 
 <x-filament::layouts.base :title="$title">
-    <div class="filament-app-layout flex w-full min-h-screen overflow-x-hidden">
+    <div class="filament-app-layout flex w-full min-h-screen overflow-x-clip">
```

Comment:
What does this fix? Curious. 😊

Reply:
It's a quirk with sticky. To be sticky the parent element has to be overflow hidden. Since the main content elements aren't directly inside the app layout it fixes that problem allowing anything inside the scrollable area to have the same perceived top to stick too. Functionality it's the same as overflow hidden.